packages/devtools-reps/src/object-inspector/index.js

```diff
@@ -24,18 +24,21 @@ const {
 
 const {
   getChildren,
+  getParent,
   getValue,
-  isDefault,
+  nodeIsDefault,
   nodeHasAccessors,
   nodeHasProperties,
   nodeIsMissingArguments,
   nodeIsOptimizedOut,
   nodeIsPrimitive,
 } = require("./utils");
 
-type Grip = {
-  actor: string
-};
+import type {
+  ObjectInspectorItemContentsValue,
+  ObjectInspectorItemContents,
+  ObjectInspectorItem,
+} from "./types";
```

Comment:
these names strike me as very verbose, especially when they're mostly going to be used as internal properties.

When you need verbosity you can always do


`import {Item as ObjectInspectorItem} from "../object-inspector"`